src/main/java/lambdastreams/task2/DisplayOfTheTask.java

```diff
@@ -0,0 +1,36 @@
+package lambdastreams.task2;
+
+import java.util.List;
+
+public class DisplayOfTheTask {
+    public static void main(String[] args) {
+        Book book1 = new Book("myBook", 5);
+        Book book2 = new Book("myBook", 5);
+
+        List<Book> listOfRandomBooks = Book.createListOfBooks(10);
```

Comment:
It's optional, but I'd still recomment to create Test class in the test package and write tests for each method (task) instead of running all of them in the main() method

Reply:
Done